frontend/packages/dev-console/src/components/import/serverless/__tests__/serverless-utils.spec.ts

```diff
@@ -50,3 +58,111 @@ describe('serverless-utils', () => {
     expect(defaultscaleUnit1).toBe('s');
   });
 });
+
+describe('hasOtherKsvcDomainMappings', () => {
+  it('should return false if the domain mapping is empty', () => {
+    expect(hasOtherKsvcDomainMappings([])).toBe(false);
+  });
+
+  it('should return false if the other ksvc info is not present in domain mapping', () => {
+    expect(hasOtherKsvcDomainMappings(['abc.org', 'example.org'])).toBe(false);
+  });
+
+  it('should return true if the other ksvc info is present in domain mapping', () => {
+    expect(hasOtherKsvcDomainMappings(['abc.org', 'example.org', 'domain.ksvc1 (ksvc1)'])).toBe(
+      true,
+    );
+  });
+});
+
+describe('removeDuplicateDomainMappings', () => {
+  it('should return empty arary if the domain mapping empty or null', () => {
+    expect(removeDuplicateDomainMappings(null, null)).toEqual([]);
+    expect(removeDuplicateDomainMappings(['a'], null)).toEqual([]);
```

Comment:
ideally, this should return `['a']`. WDYT?

Reply:
added a additional test for it.